src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article data-qa="card">
+      <div class="main-container">
+        <img
+          class="main-container__product-img"
+          src="./images/imac.jpeg"
+          alt="product"
+        />
+        <div class="main-container__description">
+          <p class="main-container__apple-model">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="main-container__product-code">Product code: 195434</p>
+        </div>
+        <div class="main-container__rating-of-product">
+          <div class="main-container__stars main-container__stars--4">
+            <div class="main-container__stars-star"></div>
+            <div class="main-container__stars-star"></div>
+            <div class="main-container__stars-star"></div>
+            <div class="main-container__stars-star"></div>
+            <div class="main-container__stars-star"></div>
+          </div>
```

Comment:
Try to use `ul` and `li` in this case. 
When we have duplicate lines best way use lists)